### DISCUSSION TITLE
Save to drive and network

### DISCUSSION THREAD

**Discussion by brianlangseth-NOAA** [pfmc-assessments/canary_2023#49] [Q&A]
@okenk I recall we had this discussion before but I couldn't find it in my email- so Im moving it here so as to be more accessible. We can discuss tomorrow during our call. 

In working up catches Im saving processed data onto my local "data" folder in our repo (which isn't pushed because of .gitignore). I haven't been consistent about whether to also save to the "data" folder in our google drive, which is the only way you would currently see it. 

I see a few standard practice choices.
1. We could remove the gitignore on the "data" folder on our local computers. We would have to ensure we only post public data there. At time of archiving, that folder would be the where we would access the files to moves to the archives. Thus we wouldn't have to save to the drive for us both to access, just save to our local folder. Lingcod did this. 
2. Upload to both our local "data" folder as well as the drive folder so we each have all the most updated material in the drive. At time of archiving, the google drive folder location would be where we would access the files to move to the archives. Seems inefficient though. 
3. (Implicit status quo decision) Maintain a mix of saving to local and saving to drive. At time of archiving, we would then have to spend more time to ensure we've archived all folders.

**Comment by brianlangseth-NOAA** [comment]
In thinking more about it, the issue of archiving should not be an issue in actuality. Files in any "data" folder probably wouldn't even need to be archived. That would be for the files in "data-raw". Thus the above discussion comes down to how we want to "share" processed data files.

**Comment by okenk** [comment]
Thanks for checking, I don't have a super strong preference on this. So whatever you would like! I need to clean up my repo a little and then will finally push the model bridged to SS3 3.30.

**Comment by brianlangseth-NOAA** [comment]
We selected option 1 (commit fe0afc6). Closing
